.github/workflows/loadtest.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
         with:
           path: ./assured-ledger
           repository: insolar/assured-ledger
+          ref: master
```

Comment:
why? its master by default in v2

Reply:
Sometimes we need to run tests on a custom branch. I always forget how to do it :D